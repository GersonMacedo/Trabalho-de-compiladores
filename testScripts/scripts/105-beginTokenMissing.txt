! -s SYNTACTIC
! should complain about missing begin token
program p ;
end .