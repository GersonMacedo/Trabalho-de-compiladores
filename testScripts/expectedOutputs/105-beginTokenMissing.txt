
Performing up to syntactic analysis at testScripts/scripts/105-beginTokenMissing.txt
Parser::Expected 'var' or 'begin' but 'end' was found at line 4 column 1
